AndriiTsurkov/HW3/1.py

```diff
@@ -0,0 +1,25 @@
+python_philosophy_str = "If the implementation is hard to explain, it's a bad idea."
+
+ocurency_better = 0
+ocurency_never  = 0 
+ocurency_is     = 0
+
+#python_philosophy_list = python_philosophy_str.split()
+
+for word in python_philosophy_str.lower().replace('.', '').replace(',', '').split():
+
+    if word == "better":
```

Comment:
I recommend using the count() method
python_philosophy_str.count("better")
etc.